src/components/Playlist.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import styles from '../styles/Playlist.module.css';
+import Track from '../components/Track.jsx';
+import { obtainUserId, addToPlaylist } from '../Constants/httpRequests.js';
+
+export default function Playlist({ trackUris, tracks, onRemoveResult }) {
+    const [playlistTitle, setPlaylistTitle] = useState('');
+    const [submissionErrorMessage, setSubmissionErrorMessage] = useState('');
+    const [savedSuccessfullyMessage, setSavedSuccessfullyMessage] = useState('');
+
+    const handlePlaylistTitleChange = (e) => {
+        setPlaylistTitle(e.target.value);
+        setSubmissionErrorMessage('');
+    };
+
+    const handleKeyDown = (e) => {
+        if (e.key === 'Enter') {
+            e.preventDefault();
+            handleSavePlaylistToSpotify();
+        }
+    };
+
+    const handleSavePlaylistToSpotify = () => {
+        if (!playlistTitle && (tracks.length === 0)) {
+            setSubmissionErrorMessage('Please add tracks and playlist name before saving');
+            
+            // Clear the error message after 5 seconds
+            setTimeout(() => {
+                setSubmissionErrorMessage('');
+            }, 5000);
+
+            return;
+        };
+        
+        if (tracks.length === 0) {
+            setSubmissionErrorMessage('Please add tracks to playlist before saving');
+       
+            // Clear the error message after 5 seconds
+            setTimeout(() => {
+                setSubmissionErrorMessage('');
+            }, 5000);
+
+            return;
+        };
+        
+        if (!playlistTitle) {
+            setSubmissionErrorMessage('Please add playlist name before saving');
+            
+            // Clear the error message after 5 seconds
+            setTimeout(() => {
+                setSubmissionErrorMessage('');
+            }, 5000);
+
+            return;
+        };
+        
+        // Obtain userId
+        obtainUserId();
```

Comment:
Can you remove this call?

Reply:
Yep. Not sure why this is here in the first place. Deleted.